ioctl/client.go

```diff
@@ -70,13 +70,16 @@ type (
 		Insecure bool
 	}
 
+	// Language type used to enumerate supported language of ioctl
+	Language int
+
 	client struct {
 		cfg       config.Config
 		conn      *grpc.ClientConn
 		cryptoSm2 bool
 
 		// TODO: merge into config
-		lang config.Language
+		lang Language
```

Comment:
I thought we need to move `lang` into `config.Config`.

Reply:
yes , i will create a new pr